test/gtest/ucs/test_time.cc

```diff
@@ -116,6 +116,7 @@ UCS_TEST_F(test_time, timerq) {
         }
         EXPECT_EQ(0u, counter1);
         EXPECT_NEAR(test_time / interval2, counter2, 1);
+        EXPECT_LE(ucs_timerq_min_interval(&timerq), UCS_TIME_INFINITY);
```

Comment:
at least LT, because current check is always true - ucs_timerq_min_interval returns ucs_time_t (unsigned long) and UCS_TIME_INFINITY is ULLONG_MAX